src/iap/modules/auth/components/signup-form/signup-form.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="iap-signup-form">
+	<h1 class="iap-signup-form__header">Account creation</h1>
+	<form
+		class="iap-signup-form__form"
+		[formGroup]="signupForm"
+		(ngSubmit)="onSubmit()"
+	>
+		<section class="iap-signup-form__login iap-input-field">
+			<label class="iap-input-field__label">Email address</label>
+			<input
+				class="iap-input-field__input"
+				formControlName="login"
+			/>
+			<div
+				class="iap-input-field__errors"
+				*ngIf="signupForm.controls['login'] | isControlInvalid"
+			>
+				<h4 *ngIf="signupForm.controls['login'].errors?.['login']">Field is required!</h4>
```

Comment:
Why didn't you do it it this way?
```suggestion
				<h4 *ngIf="signupForm.controls.login.errors?.login>Field is required!</h4>
```